src/lib/db/createStore.ts

```diff
@@ -1441,7 +1477,17 @@ async function prepareStorage(
   }, {})
   const populatedTagMap = entries(tagMap).reduce<ObjectMap<PopulatedTagDoc>>(
     (map, [name, tagDoc]) => {
-      map[name] = { ...tagDoc, name, noteIdSet: new Set() }
+      const tagColor = tagDoc.data.color
+        ? typeof tagDoc.data.color == 'string'
+          ? tagDoc.data.color
+          : undefined
+        : undefined
+      map[name] = {
+        ...tagDoc,
+        name,
+        color: tagColor,
```

Comment:
We can access color directly from `tagDoc.data.color` I don't see any benifits

Reply:
True, a mistake! Fixed now!